client/src/Protected.js

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+import { Outlet } from "react-router";
+import GetUser from "./data/GetUser";
+
+export default function Protected() {
+  const user = GetUser();
+  if (user) console.log(user.isAdmin);
```

Comment:
It is recommended to remove all the console.log from the code before merging them into your main code.

Reply:
removed!